chapter-7-continuous-integration/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# This Dockerfile is used to build an image containing basic stuff to be used as a Jenkins slave build node.
+FROM ubuntu:bionic
+MAINTAINER Ervin Varga <ervin.varga@gmail.com>
+MAINTAINER Sylvain Maucourt <smaucourt@gmail.com>
+
+# In case you need proxy
+#RUN echo 'Acquire::http::Proxy "http://127.0.0.1:8080";' >> /etc/apt/apt.conf
+
+# Add locales after locale-gen as needed
+# Upgrade packages on image
+# Preparations for sshd
+RUN apt-get -q update &&\
+    apt-get install -y locales
+RUN locale-gen en_US.UTF-8 &&\
+    DEBIAN_FRONTEND="noninteractive" apt-get -q upgrade -y -o Dpkg::Options::="--force-confnew" --no-install-recommends &&\
+    DEBIAN_FRONTEND="noninteractive" apt-get -q install -y -o Dpkg::Options::="--force-confnew" --no-install-recommends openssh-server &&\
+    apt-get -q autoremove &&\
+    apt-get -q clean -y && rm -rf /var/lib/apt/lists/* && rm -f /var/cache/apt/*.bin &&\
+    sed -i 's|session    required     pam_loginuid.so|session    optional     pam_loginuid.so|g' /etc/pam.d/sshd &&\
+    mkdir -p /var/run/sshd
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# Install GIT (latest edition)
+RUN apt-get -q update
+RUN apt-get -q upgrade -y
+RUN apt-get -q install -y git
+
+
+# Install JDK 8 (latest edition)
+RUN apt-get -q update &&\
+    DEBIAN_FRONTEND="noninteractive" apt-get -q install -y -o Dpkg::Options::="--force-confnew" --no-install-recommends software-properties-common &&\
+    add-apt-repository -y ppa:openjdk-r/ppa &&\
+    apt-get -q update &&\
+    DEBIAN_FRONTEND="noninteractive" apt-get -q install -y -o Dpkg::Options::="--force-confnew" --no-install-recommends openjdk-8-jre-headless &&\
+    apt-get -q clean -y && rm -rf /var/lib/apt/lists/* && rm -f /var/cache/apt/*.bin
+RUN apt-get -q update
```

Comment:
Idem

Reply:
Thanks for the reviews! I have applied those today. Could you review again please?